src/components/Timeline/Publication/Actions/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Comment from '@components/Publication/Actions/Comment';
```

Comment:
**question:** can we reuse this from `Publication` folder?

We can make all those changes there to support `electedMirror` as optional props

Reply:
Done